docs/openapi.rst

```diff
@@ -303,14 +303,15 @@ page using the JS script from the script URL.
 
    Default: ``None``
 
-.. describe:: OPENAPI_SWAGGER_UI_SUPPORTED_SUBMIT_METHODS
+.. describe:: OPENAPI_SWAGGER_UI_CONFIG
 
-   List of methods for which the '*Try it out!*' feature is enabled. Should be a
-   list of lowercase HTTP methods.
+   Dictionary representing Swagger UI configuration options.  See `Swagger UI Configuration`_ for available options.
+   All JSON serializable options are supported.
```

Comment:
Are there options that are not JSON serializable?

Reply:
That message is vague.  I added it because something similar was in the Readme for flask-swagger-ui.

There are several (9, I think) options that are defined as Javascript functions and wouldn't be representable in this config structure.

I could change the message to something like: `Function options are not supported.`.  Or maybe just get rid of it?

Reply:
Let's leave it as it is. I'm fine with your wording.